src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -427,12 +427,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     DependsOnTargets="_GetRestoreProjectStyle;_GetRestoreTargetFrameworksOutput"
     Returns="@(_RestoreTargetFrameworkItems)">
 
-    <PropertyGroup>
-      <_RestoreTargetFrameworkItemsHasValues Condition=" '$(TargetFramework)' != '' OR '$(TargetFrameworks)' != '' ">true</_RestoreTargetFrameworkItemsHasValues>
```

Comment:
It's just a local one used in the below item group as a condition. It was used to restrict setting the target frameworks to net core only. 
By changing it now the target frameworks are set in the legacy case as well.